### ISSUE TITLE
请问这个WARNING影响实验结果吗？

### ISSUE THREAD

**Issue by Yegiri** [AtmaHou/FewShotTagging#4]
07/03/2020 13:19:10 - WARNING - pytorch_pretrained_bert.optimization -   Training beyond specified 't_total'. Learning rate multiplier set to 0.0. Please set 't_total' of WarmupLinearSchedule correctly.

**Comment by Ison-Laihus** [comment]
不影响的，你可以选择升级pytorch-pretrained-bert和pytorch的版本
